test/test.mk

```
@@ -24,7 +22,8 @@ mode-host   := -t $(or $(TOPOLOGY),/etc/infamy.dot)
 mode-run    := -t $(BINARIES_DIR)/qemu.dot
 mode        := $(mode-$(TEST_MODE))
 
-binaries-$(ARCH) := $(addprefix $(IMAGE)-$(ARCH),.img -disk.img .pkg)
+INFIX_IMAGE_ID := $(subst ",,$(INFIX_IMAGE_ID))
```

Please use Buildroot's `qstrip` which takes care of any leading and/or trailing whitespace:

```suggestion
INFIX_IMAGE_ID := $(call qstrip,$(INFIX_IMAGE_ID))
```

---

Sure :)